v4/weaviate/batch/objectsCreate.go

```diff
@@ -22,8 +22,8 @@ type ObjectsBatcher struct {
 }
 
 // WithObject add an object to the batch
-func (ob *ObjectsBatcher) WithObject(object *models.Object) *ObjectsBatcher {
-	ob.objects = append(ob.objects, object)
+func (ob *ObjectsBatcher) WithObject(object... *models.Object) *ObjectsBatcher {
```

Comment:
Since you could add more than one object a better name would be WithObjects()

Reply:
I like this idea of adding `WithObjects`. I'm impartial to deprecating `WithObject`, because accepting a single object still makes sense, even though `WithObjects` can still be used with only one obj. Either way is ok IMO

Same with `WithReference(s)` below ⬇️